humanguide/layout/02-header-uses.rst

```diff
@@ -0,0 +1,18 @@
+Header uses 
+===========
+
+App header can be used to show app name, app section or page, search
+box, etc.
```

Comment:
"App headers can" or "An app header can". Instead of "etc.", consider defining more clearly what else could go in a header. Maybe ", or other globally accessible functions."